src/framework/audio/internal/worker/mixerchannel.cpp

```diff
@@ -261,3 +266,10 @@ void MixerChannel::notifyNoAudioSignal()
 
     m_audioSignalNotifier.notifyAboutChanges();
 }
+
+bool MixerChannel::playNotesOnMutedTracksWhenEditing() const
+{
+    // We are not muted if we are Editing and we want to play notes even if mixer sound is off
+    return !playbackController()->isPlaying()
```

Comment:
Unfortunately, this is incorrect for many reasons:

1. this creates bad (circular) dependencies between the audio and playback modules. The audio module must not know anything about the playback module, as the latter is a higher-level one
2. this code is thread-unsafe
3. constantly accessing these settings may cause performance degradation